app/src/main/java/com/asfoundation/wallet/verification/intro/VerificationIntroPresenter.kt

```diff
@@ -23,19 +24,37 @@ class VerificationIntroPresenter(private val view: VerificationIntroView,
                                  private val data: VerificationIntroData,
                                  private val analytics: VerificationAnalytics) {
 
+  private var currentError: ErrorState? = null
+
   companion object {
+    private const val CURRENT_ERROR_KEY = "current_error"
```

Comment:
Shouldn't this be packate private instead of private to avoid the creation of a synthetic accessor?